src/common/components/vanilla-modules/icons/index.js

```diff
@@ -0,0 +1,202 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import styles from '../../../style/vanilla/css/icons.css';
+
+// Creates icon style class name
+const iconStyle = (icon = '') => {
+  icon = icon ? '--' + icon : '';
```

Comment:
`icon` name here is a bit confusing when it's used for other things (size, color), so maybe a `modifier` would work better as this variable name?